hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/dsl/annotations/FungibleToken.java

```diff
@@ -60,4 +60,10 @@
      * @return the initial supply of the token
      */
     long initialSupply() default 100;
+
+    /**
+     * The max supply of the token.
+     * @return the max supply of the token
+     */
+    long maxSupply() default 0;
```

Comment:
(This is the first time I've seen the syntax that you can declare a default method without the `{ return xxx;}` syntax.  Can't find a single example on the net of this!  Will next go and find the Java spec to look up the actual grammar ...)